### PR TITLE
fix(disp): change all disp_buf to draw_buf

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
### Description of the feature or fix

and remove LV_HOR_RES_MAX since it doesn't exist anymore

### Checkpoints
- [X] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update CHANGELOG.md
- [ ] Update the documentation

### CODE REVIEW

File: examples/arduino/LVGL_Arduino.ino [hunk 1]
```diff
@@ -81,7 +81,7 @@ void setup()
     uint16_t calData[5] = {275, 3620, 264, 3532, 1};
     tft.setTouch(calData);
 
-    lv_disp_buf_init(&disp_buf, buf, NULL, LV_HOR_RES_MAX * 10);
+    lv_disp_buf_init(&disp_buf, buf, NULL, 320 * 10);
```
Comment (embeddedt) [inline examples/arduino/LVGL_Arduino.ino]: How about `screenWidth` instead?

Comment (xiaoxiang781216) [reply examples/arduino/LVGL_Arduino.ino]: Done.

**Comment by kisvegabor** [comment]
Thank you!

**Comment by embeddedt** [comment]
How did GitHub Actions manage to merge a PR? :laughing: Visual bug only?

**Comment by kisvegabor** [comment]
I pulled the changes, merged them to master manually to let git recognize the renamed folders and pushed to master. 
The action then merged master to dev. So GitHub could either use my push or the action to close the PR. It hasn't chosen wisely :laughing:

**Pull request merged**